components/fieldgroups/profile/profile-healthcare-member.schema.json

```diff
@@ -0,0 +1,367 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile-healthcare-member",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Healthcare Member Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["healthcare_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Details of a person/member/subscriber who may be waiting for a care or may be receiving care or may have already received care.",
+  "definitions": {
+    "member-details": {
+      "properties": {
+        "xdm:member": {
+          "title": "Member",
+          "description": "Information about the member.",
+          "type": "object",
+          "properties": {
+            "xdm:ID": {
+              "title": "Member ID",
+              "type": "string",
+              "description": "Unique Identifier of the patient/member."
```

Comment:
Unique identifier of the patient/member.

Reply:
We are removing this so, this change won't be required